tests/ops/test_simple_gla.py

```
@@ -254,6 +252,153 @@ def test_chunk_varlen(
+
+
+@pytest.mark.parametrize('N', test_b_list)
+@pytest.mark.parametrize('T', test_t_varlen_list)
+@pytest.mark.parametrize('H', test_h_list)
+@pytest.mark.parametrize('D', test_d_list)
+@pytest.mark.parametrize('dtype', [torch.float16])
+@pytest.mark.skipif(
+    os.getenv('SKIP_TEST_CHUNK_VARLEN') == '1',
+    reason='Skipping test_chunk_varlen because SKIP_TEST_CHUNK_VARLEN is set'
+)
+def test_fused_chunk_varlen(
```

Pylint (3.3.7)

[refactor] Too many local variables (27/15)

(R0914)